examples/step-44/step-44.cc

```diff
@@ -844,29 +841,24 @@ namespace Step44
 
     void determine_component_extractors();
 
+    // Apply Dirichlet boundary conditions on the displacement field
```

Comment:
The comment does not match the name of the function, do you apply constraints to the displacement or do you create constraints?

Reply:
True. It was a lazy cut and paste. I've changed the comment to "Create Dirichlet constraints for the incremental displacement field".